data/pages-content/sprints.md

```diff
@@ -207,6 +207,24 @@ Writing PEPs for the given Issues below. The organizer has never written [a PEP]
    - [Not Operator](https://github.com/python/typing/issues/801)
    - [copy_kwargs function](https://stackoverflow.com/questions/71968447/python-typing-copy-kwargs-from-one-function-to-another)
 
+### Pytorch - Tensors and Dynamic neural networks in Python
+
+- Number of people: ~10 (1 organizer)
+- Room: RB 104
```

Comment:
Please update the room occupancy list at the top of the sprint list.

Reply:
I've updated the room capacity